server/package.json

```diff
@@ -36,7 +36,9 @@
     "js-yaml": "^4.1.0",
     "json-refs": "^3.0.15",
     "morgan": "^1.10.0",
+    "multer": "^1.4.5-lts.1",
     "pg": "^8.11.2",
+    "react-dropzone": "^14.2.3",
```

Comment:
Is `react-dropzone` necessary on the server side?

Reply:
no, removed